extra_docs/quickstart.md

```diff
@@ -245,100 +270,130 @@ Filtering conditions can be nested at any level.
 
 ``` Javascript
 [..]
-let flatFilter = field("first_name").isEqualTo("Tom")
-                 .or( field("first_name").isEqualTo("Dick"))
-                 .or( field("first_name").isEqualTo("Harry"));
-
-let nestedFilter = field("first_name").isEqualTo("Tom")
-                   .or( field("first_name").isEqualTo("Dick")
-                        .and( field("middle_name").isEqualTo("Harry")));
-
-let anotherNestedFilter = field("first_name").isEqualTo("Tom")
-                   .or( field("first_name").isEqualTo("Dick")
-                        .and( field("middle_name").isEqualTo("Harry")
-                              .or( field("middle_name").isEqualTo("Larry"))));
+// Filtering in a flat way
+const flatFilter = field("first_name")
+  .isEqualTo("Tom")
+  .or(field("first_name").isEqualTo("Dick"))
+  .or(field("first_name").isEqualTo("Harry"));
+
+// Filtering with one nested level
+const isDickHarry = field("first_name")
+  .isEqualTo("Dick")
+  .and(field("middle_name").isEqualTo("Harry"));
+
+const nestedFilter = field("first_name")
+  .isEqualTo("Tom")
+  .or(isDickHarry);
+
+// Filtering with multiple nested levels
+const isDick = field("first_name").isEqualTo("Dick");
```

Comment:
TBH I think this example is pretty confusing...
You are search for people called both 'Dick' _and_ 'Harry', how is this possible (yes when I look more carefully I see it is different fields...)
Maybe search for name and age (for example) to make the example more clear (without studying it)?

Reply:
indeed, changed the examples.